Doc/library/csv.rst

```diff
@@ -88,7 +88,7 @@ The :mod:`csv` module defines the following functions:
 
    Return a writer object responsible for converting the user's data into delimited
    strings on the given file-like object.  *csvfile* can be any object with a
-   :func:`write` method.  If *csvfile* is a file object, it should be opened with
+   :meth:`~io.TextIOBase.write` method.  If *csvfile* is a file object, it should be opened with
```

Comment:
@serhiy-storchaka Is this the correct one for `write` in this case?

Reply:
Good question. I mulled that over a bit before using `TextIOBase`, eventually deciding that since CSV files are supposed to be opened as plain text files since Python 3, that was the most reasonable link destination.

Reply:
You can also explicitly mention that `write()` should accept a string argument (not bytes). And the same for the reader.